packages/realm/bindgen/.eslintrc.json

```diff
@@ -0,0 +1,16 @@
+{
+  "root": true,
```

Comment:
While I understand and appreciate you want this to be an example for other teams, I think having this a root config is not really in the best interest of this team. We'd want a change to the eslint config in the repo root to have an effect the code in this package.